index.html

```diff
@@ -5,32 +5,35 @@
     <title> About Rylee </title>
   </head>
   <body>
-    <h1>About Me Rylee Andrews</h1>
-    <p>My name is Rylee Andrews and I am originally from Montana. Growing up my parents owned a t-shirt printing shop, so I developed a love for clothes. So my senior year I started my own clothing line and have been working on it since. Outside of working on my clothing I ski and snowboard and cliff jump during the summer. I'm a huge raver and you will most likely see me at every USC event. My favorite food is sushi and my favorite roll is the spider roll. I have been to Canada and the Bahamas and I dream of seeing Thailand.</p>
-    <h2>How I Got Into Coding</h2>
-    <p>I got into coding because my buddy Sam, who also attended code fellows, told me to look into getting into code fellows. So I started learning coding online with bucky. I knew I wanted to further pursue this right after I finished the first tutorial on html and css I absolutely loved it! My favorite language by far is javascript just becuase of the many possibilites it gives you.</p>
-    <h2>Future goals</h2>
-    <p>My future goals are to become a full stacked javascript developer and be able to completely design ecommerce sites for all my future business ventures. Like everyone's goal we all want to be rich but I actually work for it so a nice house on the ocean is in my future.</p>
+    <h1> About Me Rylee Andrews </h1>
+    <p> My name is Rylee Andrews and I am originally from Montana. Growing up my parents owned a t-shirt printing shop, so I developed a love of clothes. So my senior year I started my own clothing line and have been working on it since. Outside of working on my clothing I ski and snowboard and cliff jump during the summer. I also am a huge raver and most likely see me at every USC event. </p>
+    <h2> Future goals </h2>
+
+    <p> My future goals are to become a full stacked javascript developer and be able to completely design ecommerce sites for all my future business ventures. Like everyone's goal we all want to be rich but I actually work for it so a nice house on the ocean is in my future. </p>
     <ul>
-      <li>Big house</li>
-      <li>Trophy wife</li>
-      <li>Lamborghini</li>
-      <li>A condo on the mountain so I could ski everyday</li>
-      <li>Master developer</li>
-    </ul>
-    <h1>Top Ten Places I want to visit</h1>
-    <ul>
-      <li>Thailand</li>
-      <li>Japan</li>
-      <li>Italy </li>
-      <li>Greece</li>
-      <li>Argetina</li>
-      <li>Peru</li>
-      <li>Chile</li>
-      <li>Australia</li>
-      <li>Bora Bora</li>
-      <li>Virgin Islands</li>
+      <li> Big house </li>
+      <li> Trophy wife </li>
+      <li> Lamborghini </li>
+      <li> A condo on the mountain so I could ski everyday </li>
+      <li> Master developer </li>
     </ul>
+
```

Comment:
it's a good idea to keep your white-space consistent, it makes your work more readable and is considered good-practice.